.vsts-ci.yml

```diff
@@ -99,10 +99,10 @@ phases:
     workingDirectory: node
 
 #################################################
-- phase: build_on_osx
+- phase: macOS
 #################################################
-  displayName: (vsts-task-lib) osx
-  queue: Hosted Mac Internal
+  condition: and(succeeded(), eq(variables.os, 'Darwin'))
+  queue: Hosted macOS Preview
```

Comment:
i might need to switch this back to "Hosted Mac Internal". Every public account has "Hosted macOS Preview", but we only have 1 agent in that pool so i'm not sure what is going on. i'll check with mms folks tomorrow.